services/scorecard/scorecard.service.js

```diff
@@ -0,0 +1,41 @@
+import Joi from 'joi'
```

Comment:
We should use a more explicit, descriptive name that represents the actual upstream service. "Scorecard" is very generic and we can't just assume/assign that generic term to mean OSSF's score card.

Please rename the directory, files, and class accordingly